internal/resource/config/token_processor_to_token_generator_mapping_resource.go

```diff
@@ -86,6 +75,8 @@ func (r *tokenProcessorToTokenGeneratorMappingsResource) Schema(ctx context.Cont
 		},
 	}
 	AddCommonSchema(&schema)
+	AddCustomId(&schema, false, false,
+		"The id of the Token Processor to Token Generator mapping. This field is read-only and is ignored when passed in with the payload.")
```

Comment:
If this field is ignored do we even need a custom_id attribute on this resource?